queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -369,37 +370,41 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
 
                 # update affiliation for new business
                 if filing_core_submission.filing_type != FilingCore.FilingTypes.CONVERSION:
-                    business_profile.update_affiliation(business, filing_submission)
+                    business_profile.update_affiliation(business, filing_submission, flags)
 
                 name_request.consume_nr(business, filing_submission, flags=flags)
-                business_profile.update_business_profile(business, filing_submission)
+                business_profile.update_business_profile(business, filing_submission, flags=flags)
                 await publish_mras_email(filing_submission)
             else:
                 # post filing changes to other services
-                for filing_type in filing_meta.legal_filings:
-                    if filing_type in [
-                        FilingCore.FilingTypes.ALTERATION,
-                        FilingCore.FilingTypes.CHANGEOFREGISTRATION,
-                        FilingCore.FilingTypes.CORRECTION,
-                        FilingCore.FilingTypes.DISSOLUTION,
-                        FilingCore.FilingTypes.PUTBACKON,
-                        FilingCore.FilingTypes.RESTORATION
-                    ]:
-                        business_profile.update_entity(business, filing_type)
-
-                    if filing_type in [
-                        FilingCore.FilingTypes.ALTERATION,
-                        FilingCore.FilingTypes.CHANGEOFREGISTRATION,
-                        FilingCore.FilingTypes.CHANGEOFNAME,
-                        FilingCore.FilingTypes.CORRECTION,
-                        FilingCore.FilingTypes.RESTORATION,
-                    ]:
-                        name_request.consume_nr(business,
-                                                filing_submission,
-                                                filing_type=filing_type,
-                                                flags=flags)
-                        if filing_type != FilingCore.FilingTypes.CHANGEOFNAME:
-                            business_profile.update_business_profile(business, filing_submission, filing_type)
+                if not flags.is_on('enable-sandbox'):
+                    for filing_type in filing_meta.legal_filings:
+                        if filing_type in [
+                            FilingCore.FilingTypes.ALTERATION,
+                            FilingCore.FilingTypes.CHANGEOFREGISTRATION,
+                            FilingCore.FilingTypes.CORRECTION,
+                            FilingCore.FilingTypes.DISSOLUTION,
+                            FilingCore.FilingTypes.PUTBACKON,
+                            FilingCore.FilingTypes.RESTORATION
+                        ]:
```

Comment:
Just added this block to skip post-processing for non-business creation filings altogether.  We want to avoid this as you may end up updating auth entity info for a prod based business.